edge/pkg/edgehub/common/http/http_test.go

```diff
@@ -26,8 +26,8 @@ import (
 	"strconv"
 	"testing"
 
-	"github.com/kubeedge/kubeedge/pkg/common/util"
-	common "github.com/kubeedge/kubeedge/pkg/edgehub/common"
+	"github.com/kubeedge/kubeedge/edge/pkg/common/util"
+	common "github.com/kubeedge/kubeedge/edge/pkg/edgehub/common"
```

Comment:
please move this section after line: "github.com/stretchr/testify/assert"

Reply:
fixed